h/static/scripts/widget-controller.js

```diff
@@ -219,12 +221,10 @@ module.exports = function WidgetController(
     var group = annotationUI.hasSelectedAnnotations() ?
       null : groups.focused().id;
 
-    for (var i=0; i < urls.length; i++) {
-      _loadAnnotationsFor(urls[i], group);
-    }
+    _loadAnnotationsFor(searchUris, group);
```

Comment:
Since `_loadAnnotationsFor` is now outside of a loop, it gets called here even if `searchUris` is empty. The result when I tested this locally on a PDF is that the client ended up trying to fetch all public annotations. The call to `_loadAnnotationsFor` needs to be moved inside the `if (searchUris.length) {...}` check below.